zadanie6/views/next.erb

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+    <meta name="theme-color" content="#000000">
+    <link href="/styles.css" rel="stylesheet" type="text/css">
+    <title>Game of Life</title>
+  </head>
+  <body>
+    <form method="post" action="/next">
+      <input type="hidden" name="alive_cells" value= '<%= @alive_cells.to_json %>'>
+      <table>
+        <% 30.times do |row| %>
+          <% row+= 1 %>
+          <tr>
+            <% 50.times do |column| %>
+              <% column+=1 %>
+              <% if @alive_cells.include?("#{row}_#{column}") %>
```

Comment:
To rozwiązanie działa i jest poprawne, ale jestem fanem tak "głupich" (w sensie braku logiki aplikacji) widoków jak to tylko możliwe, więc rozważyłbym umieszczenie wszystkich komórek w hashu którego kluczem byłyby numery wierszy i kolumn a wartością: `dead` lub `alive` i wstawianie jej bezpośrednio w htmlową klase.